### ISSUE TITLE
data availability

### ISSUE THREAD

**Issue by bgriffen** [AusClimateService/hazards-drought#46]
Where might I be able to download these maps/data? Very keen.

Also, how might it be different to datasets I can export here: https://awo.bom.gov.au/

**Comment by dahoal2** [comment]
Thanks for your interest in our data. Unfortunately, the data is not yet openly available as we are still finalising the technical documentation before we release it.
In contrast to the AWO data which is based on CMIP5 global climate models, the drought data here is based on CMIP6, with the exception of the aridity index which uses the same data as found in the AWO.

**Comment by bgriffen** [comment]
OK, would you have at time frame on the release? A notebook generating each baseline data asset would be helpful as well as someone who isn't familiar with filenames and paths, it's a bit difficult to navigate asset is which. Thank you for the efforts!

**Comment by dahoal2** [comment]
No, unfortunately, I don't have a time frame for you. This is a national effort with a lot of turning wheels. The filenames and paths are complaint with CORDEX convention and referring to data on NCI that has not been released publicly yet.
